platforms/cpu/include/CpuNonbondedForce.h

```diff
@@ -203,8 +203,8 @@ class CpuNonbondedForce {
         float inverseRcut6Expterm;
         std::atomic<int> atomicCounter;
 
-        static const float TWO_OVER_SQRT_PI;
-        static const int NUM_TABLE_POINTS;
+        static constexpr float TWO_OVER_SQRT_PI = 1.128379167095513;
```

Comment:
I think the original version of this was better: `TWO_OVER_SQRT_PI = (float) (2/sqrt(PI_M))`.  It's clearer, and constants embedded as long strings of digits are a common source of errors.

Reply:
sqrt isn't a constexpr function though. The code generated from the original was doing a runtime conversion from float to vector where it was used.

Reply:
No, it was doing a conversion at startup time when the field was initialized.

Reply:
I realise that the initialisation did the first conversion, but on SSE and AVX also did a float-load, followed by a broadcast at each point of use of that value too. On IA at least, it is better to use a broadcast-load instruction directly. Switching to constexpr fixed that.

Doesn't really matter though, as for the moment I have backed out this change as it was breaking the MAC build.

Reply:
Maybe create the fvec version of it inside the method, before the start of the loop that uses it?